src/vivarium_cluster_tools/cli.py

```diff
@@ -21,7 +21,9 @@ def psimulate():
 @click.option('--result-directory', '-o', default=None,
               help='The directory to write results to. A folder will be created in this directory with the same name '
                    'as the configuration file.')
-@click.option('--project', '-P', type=click.Choice(['proj_cost_effect', 'proj_csu', 'proj_cost_effect_diarrhea']), default='proj_cost_effect',
+@click.option('--project', '-P', type=click.Choice(['proj_cost_effect', 'proj_cost_effect_diarrhea',
```

Comment:
now that I see this in a PR instead of me just hacking an extra project in to make diarrhea work, maybe we should have a global list for the projects? Just so that run and restart don't end up with a mismatched list?

Reply:
I don't have an issue with that, it fits the bill of a global variable

Reply:
Pull it out into a globals.py file so we can use it in the runner as well.